### PR TITLE
8.3.0 Release

### PR DESCRIPTION
**Pull request by UTDZac**
If/when you have time, can you double-check your Quickload stuff still work? I'm fairly confident the small checks I did to load in the proper attempts count should work fine, but also worried I missed something. Also try changing to another quickload setup you've used in the past, such as switching games or challenge modes (rnqs settings file). Hopefully the attempts counter stuff updates itself properly and the quickload screen UI properly reflects the changes. 

### Release Notes
- You can now look up all possible [Randomized Evos](https://user-images.githubusercontent.com/4258818/266699708-174a0b8a-281d-459a-bc9e-129c8c66f18c.png) by clicking on a Pokémon's icon; percentages shown are estimates (Shoutout to brdy and Harkenn)
- A simple [Coverage Calculator](https://user-images.githubusercontent.com/4258818/266700515-e001d5ec-f9bf-4c6a-99c6-b01609e2c31a.png) has been added under Extras > Tools
- Fixed a critical bug where changing Quickload settings would incorrectly overwrite your attempts count. In addition, the Quickload screen setup UI has been updated to help show what settings you are using
- Fixed a bug where manually loading saved Tracker notes would fail to actually load them into the Tracker
- Fixed a bug where the "Congratulations" screen would change to "GameOver" after leaving the Hall of Fame
- Fixed a bug where the drop-down list of Themes would grow in size
- For a full list of new features, changes and bug fixes, check out the **[Version Changelog](https://github.com/besteon/Ironmon-Tracker/wiki/Version-Changelog)**

#### Quickload Related changes
- 4fc34e731e2ea1925bf40f77fe2d4d2f8940e0bd Changing quickload properly updates attempts
- edc2d36d19cfd1894e1861441cf3eab7141ebea5 Fix Attempts file incorrect reference
- 701ce3bd22f0a1d4075cab43664f041bdc147d82 Update attempts # when swapping methods
- 6f5f450452a32413b147c552e996bb610a92e413 Quickload Screen UI updates

#### Other changes not included in a PR
- 87b419da8e275538a315c15e47b70f07d904bd71 Add in remaining base friendship values
- e39876426a16d9e91823ec4d01a331272d2d5094 Load Tracked Data now force imports
- 13e8535322c88570e4f311f75c37f9ed8661ad15 Add checks for Shiny and PokéRus
- 5d4e74c90cbd35410dec6033cca1c9a741cb6bd2 Fix CONGRATS screen from changing after win
- bc6bcd85ec4d3f3ea153ece4b7f07b3951aeba69 Ball Picker UI updates
- 8ed63e93d290050a7ce725fcf99bd9c534b6a931 Fix Theme Presets clear out on quickload
- eed28358a8af844af327ba9e074dfd5965050274 MGBA: Fixed missing instructions for quickload rom generation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,9 +54,9 @@ We'd ideally like to support all non-English versions if we can, progress update
 
 ## Latest Changes
 
-### Walking Pals Interactive Icon Set
+### Randomized Evos and Coverage Calculator
 
-![Image showing the new walking pals icon set. It shows a Quagsire standing in a walking-position on the main Tracker screen. The next image shows the same Quagsire asleep with its mouth open.](https://user-images.githubusercontent.com/4258818/265281719-455fdc67-8e65-460a-9eff-e536ffba7928.png)
+![A screen showing the possible random evolutions for Bublasaur; twelve visible on the current page.](https://user-images.githubusercontent.com/4258818/266699708-174a0b8a-281d-459a-bc9e-129c8c66f18c.png)![A screen showing the new coverage calculator, with four move types chosen and the calculated coverage displayed below.](https://user-images.githubusercontent.com/4258818/266700515-e001d5ec-f9bf-4c6a-99c6-b01609e2c31a.png)
```
Comment (jtigues) [inline README.md:59]: Incredibly minor, but 'Bublasaur' is misspelled

Comment (UTDZac) [reply README.md]: Fixed: 713f689141877fb0dadde212f297c42b6b1e1971

**Review by Fellshadow — CHANGES_REQUESTED** [review]
Code looks fine to me, but noticed an issue with the quickload which i've discussed in the discord. Done what I think will fix it in #413 so will want to double-check and merge that in first.

File: ironmon_tracker/screens/RandomEvosScreen.lua [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+			textColor = revoInfo.perc < 0.1 and "Negative text" or RandomEvosScreen.Colors.text,
+			id = evoId,
+			ordinal = i,
+			dimensions = { width = 32, height = 32, },
+			boxColors = { RandomEvosScreen.Colors.border, RandomEvosScreen.Colors.boxFill },
+			isVisible = function(self) return RandomEvosScreen.Pager.currentPage == self.pageVisible end,
+			getIconId = function(self) return self.id, SpriteData.Types.Idle end,
+			onClick = function(self)
+				InfoScreen.changeScreenView(InfoScreen.Screens.POKEMON_INFO, self.id) -- implied redraw
+			end,
```
[the remark this answers is not in the document]
Comment (UTDZac) [reply ironmon_tracker/screens/RandomEvosScreen.lua:160]: Good note. This is expected for now. I don't currently have a good solution for tracking breadcrumbs to return back to a previous screen, especially in cases such as this where extra info ID data is required to load that screen, such as pokemon ID. For now, im okay with just leaving it with a simple "back" behavior where the current screen closes and a different screen opens.

**Comment by UTDZac** [comment]
Merged in #413 to help fix the cases where quickload setup changing results in the wrong attempts # showing by default.

**Review by Fellshadow — APPROVED** [review]
should be all good now (hopefully)

**Pull request merged**